boilerplates/react-union-boilerplate-liferay-ssr/liferay/ssr-filter/src/main/java/eu/reactunion/ssr/service/RenderingService.java

```diff
@@ -0,0 +1,53 @@
+package eu.reactunion.ssr.service;
+
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import feign.Feign;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ServiceScope;
+
+@Component(
+        immediate = true,
+        service = RenderingService.class,
+        scope = ServiceScope.SINGLETON
+)
+public class RenderingService {
+
+    private static final Log log = LogFactoryUtil.getLog(RenderingService.class);
+
+    private final RenderingServiceClient renderingServiceClient = Feign
+            .builder()
+            // TODO: Make URL configurable.
+            .target(RenderingServiceClient.class, "http://localhost:3303");
```

Comment:
This needs to be fixed, I was wainting for @raptor22fa with the changes... but we can do it in another PR 🙂